### ISSUE TITLE
Video element is not treated

### ISSUE THREAD

**Issue by iherman** [danielweck/epub3-sliderizer#4]
If I have, e.g., the following in my data file:

```
<video id="video_embed" class="fullxx" controls="controls">
<source src="videos/JavaBook.mp4" type="video/mp4" />
<source src="videos/JavaBook.webm" type="video/webm" >
<source src="videos/JavaBook.ogv" type="video/ogg"></source>
</video>
```

I would expect this to be treated like images, ie, the video files are copied to the output and the reference added to the `package.opf` file. It does not happen:-(

**Comment by danielweck** [comment]
Thanks @iherman I will look into this. At first glance, the use of multiple `source` elements instead of the `src` attribute introduces some additional complexity (parser / generator + asset handler), but as always this is just a question of time and effort ;)
Unfortunately I do not have any spare time to work on this pet project at the moment :(

**Comment by iherman** [comment]
I had a similar issue when using one video element with its own src... :-(

---

Ivan Herman
+31 641044153

(Written on my mobile. Excuses for brevity and frequent misspellings...)

**Comment by danielweck** [comment]
@iherman That's correct, support for video asset handling is missing entirely (`src` attribute and `source` elements).
